server/src/test/java/org/elasticsearch/search/aggregations/bucket/SignificantTermsSignificanceScoreIT.java

```diff
@@ -513,6 +514,7 @@ public void testScoresEqualForPositiveAndNegative(SignificanceHeuristic heuristi
                     .addAggregation(terms("class").field("class").subAggregation(significantTerms("mySignificantTerms")
                             .field("text")
                             .executionHint(randomExecutionHint())
+                            .collectMode(randomFrom(Aggregator.SubAggCollectionMode.values()))
```

Comment:
We also need a test with sub-aggregations (a `terms` aggregation under a `significant_terms` for instance) to test the `breadth_first` mode. Without sub-aggregations `depth_first` and `breadth_first` are no different.